assets/queries/openAPI/success_response_code_undefined_get_operation/query.rego

```diff
@@ -0,0 +1,22 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+	response := doc.paths[n][oper].responses
+
+	oper == "get"
```

Comment:
I think it would make more sense to just access get responses directly, something like:
`response := doc.paths[n].get.responses`, no?